### ISSUE TITLE
CSS Output for Typography

### ISSUE THREAD

**Issue by alex-nick** [reduxframework/redux-framework#3562]
Hi guys, thank you for the great plugin.

When using the typography control
```
array(
    'type'    => 'typography',
    'output' => array('h1, h2, h3'),
),
```

The **.wf-loading** selector is generated only once at the beginning of the output, that causes my typography elements, except the first one to be not displayed.
```
.wf-loading h1, h2, h3 {
    opacity: 0;
}
```
It's supposed to be prepended to all elements.
Like this:
```
.wf-loading h1, .wf-loading h2, .wf-loading h3 {
    opacity: 0;
}
```

Thank you for looking into this.

**Comment by kprovance** [comment]
Please go back and, open a new issue window, review all the text before deleting it out. and reply back with the required information.  I cannot offer assistance without it.  Thanks.

**Comment by alex-nick** [comment]
Hi, I would like to report a bug.
There is a bug in the update that launched recently.
The Typography control generates css, that prevents the typography from being displayed.

## Support Hash
Please note It's a localhost site
http://support.redux.io/?id=JhvQw9umZucMU7QRgChb6x2IgYF0lNguABE%2FG5DM0IVAOvMbIDgXeJkns6ypGmmWI8xuC9cwShDBF4izaZrRSQ%3D%3D

## Steps to reproduce
Use typography field with direct tag reference. For example: h1, h2, h3

```
array(
    'type'    => 'typography',
    'output' => array('h1, h2, h3'),
),
```


## Expected Behavior
All texts wrapped into these tags should be shown on the front end when the .wf-loading selector is prepended before each tag on the output

```
.wf-loading h1, .wf-loading h2, .wf-loading h3 {
    opacity: 0;
}
```

## Actual Behavior
The .wf-loading selector is prepended only before the first tag in tags array and therefor non of the texts is visible besides the first tag.

**Comment by kprovance** [comment]
Did it work in the last version, or is this a recently discovered thing?

**Comment by alex-nick** [comment]
It appeared in version 3.6.10 because of this fix: Fixed: #3534 – Fix invalid CSS in asycn_typography (trailing commas).

**Comment by kprovance** [comment]
Ok.  Looking into it now.  Will post back when I fix it.

**Comment by kprovance** [comment]
lol, well...this is interesting.  You are right, Redux is not prefixing each selector, only the first.  Here is the interesting part...it never has.  The only reason folks are noticing it was because the extra comma removal from the CSS began enofcing the bad CSS.  With the broken CSS, the selectors always rendered, albeit improperly.  So the CSS 'fix' revealed a bug that has always been there.  Lovely, yeah?

Okay, I can fix this.  Might take an hour or so.  But I will get this updated by the end of today on wp.org.

**Comment by alex-nick** [comment]
That's super great! Thanks again for the plugin!

**Comment by dzt99** [comment]
Oh ok it's reported. Thanks so much for the plugin.

**Comment by kprovance** [comment]
@dzt99, yes...I am coding out the fix now and will update wp.org straight away.  Sorry about this.  Sometimes user contributed fixes don;t account for every known variable, including revealing a bug we've had all along.  Blessing in disguise, actually.  I'll post back when I've updated the wp.org repo so you can get the fix.

**Comment by kprovance** [comment]
Version 3.6.11 has been pushed to this repo and to wp.org.  Sorry again, folks.
